custom/icds_reports/ucr/data_sources/dashboard/primary_private_school_form_ucr.json

```diff
@@ -0,0 +1,240 @@
+{
+  "domains": [
+    "icds-dashboard-qa",
+    "sankalp_cas",
+    "zohaib-sandbox",
+    "akshita-sandbox",
+    "sunaina-sandbox",
+    "laurence-project-1",
+    "jessica-icds-cas",
+    "marissa-test",
+    "derek-icds-sandbox",
+    "priyanka-app",
+    "shrena-dev",
+    "aparatest",
+    "reach-sandbox",
+    "reach-dashboard-qa",
+    "reach-test",
+    "icds-cas",
+    "cas-lab"
+  ],
+  "server_environment": [
+    "india",
+    "icds"
+  ],
+  "config": {
+    "table_id": "static-dashboard_primary_private_school",
+    "display_name": "Forms - Primary private school (Static) (Dashboard)",
+    "referenced_doc_type": "XFormInstance",
+    "description": "Primary private form used by dashboard to exclude children from pse eligible",
+    "base_item_expression": {
+      "type": "coalesce",
+      "expression": {
+        "type": "property_path",
+        "property_path": ["form", "case_load_person_0", "case", "@case_id"]
+      },
+      "default_expression": {
+        "type": "constant",
+        "constant": [0]
+      }
+    },
+    "configured_filter": {
+      "operator": "eq",
+      "type": "boolean_expression",
+      "expression": {
+        "datatype": null,
+        "type": "property_name",
+        "property_name": "xmlns"
+      },
+      "property_value": "http://openrosa.org/formdesigner/022E5442-0F66-430F-90CB-1E86544D5C5D"
+    },
+    "validations": [
+      {
+        "name": "submitted_by_aww",
+        "error_message": "Form not submitted by AWW",
+        "expression": {
+          "type": "boolean_expression",
+          "operator": "eq",
+          "expression": {
+            "location_id_expression": {
+              "type": "named",
+              "name": "user_location_id"
+            },
+            "type": "location_type_name"
+          },
+          "property_value": "awc"
+        }
+      }
+    ],
+    "configured_indicators": [
+      {
+        "column_id": "timeend",
+        "datatype": "datetime",
+        "type": "expression",
+        "expression": {
+          "type": "root_doc",
+          "expression": {
+            "type": "property_path",
+            "datatype": "datetime",
+            "property_path": [
+              "form",
+              "meta",
+              "timeEnd"
+            ]
+          }
+        }
+      },
+      {
+        "column_id": "state_id",
+        "datatype": "string",
+        "type": "expression",
+        "expression": {
+          "type": "ancestor_location",
+          "location_id": {
+            "type": "named",
+            "name": "user_location_id"
+          },
+          "location_type": "state",
+          "location_property": "_id"
+        },
+        "create_index": true
```

Comment:
this index isn't needed as its covered by `(state_id, timeend)`

Reply:
Ow. I did not know that. removed!